rabbit-escape-engine/src/rabbitescape/engine/behaviours/Walking.java

```diff
@@ -274,6 +274,7 @@ public boolean behave( World world, Rabbit rabbit, State state )
                 return true;
             }
             case RABBIT_TURNING_LEFT_TO_RIGHT:
+                rabbit.onSlope = false; // Intentional fall-through
             case RABBIT_TURNING_LEFT_TO_RIGHT_RISING:
```

Comment:
Possibly. Again, this is not for this change, just musing. Fortunately we've got good test coverage so can contemplate refactoring.